src/lib/update-cors-cloud.spec.ts

```diff
@@ -0,0 +1,63 @@
+import axios from 'axios';
+import MockAdapter from 'axios-mock-adapter';
+import { updateCorsConfig } from './update-cors-cloud';
+
+const mock = new MockAdapter(axios as any);
+
+describe('libUpdateCorsCloud', () => {
+  beforeEach(() => {
+    mock.onPut('/global-config/services/CorsService').reply(201, { _id: '123' });
+    mock.onGet('/json/global-config/services/CorsService/?_action=nextdescendents').reply(200, {
```

Comment:
I guess this `?_action=nextdescendents` is coming from the API swagger docs, but I wonder why you need it?

Reply:
I took this directly from the AM documentation, i mostly copied the url from there directly. Not really sure if I need it though but i'll look into it